src/components/course-card/CourseCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import IconButton from '@mui/material/IconButton'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import Divider from '@mui/material/Divider'
+
+import { styles } from '~/components/course-card/CourseCard.styles'
+import SubjectLevelChips from '../subject-level-chips/SubjectLevelChips'
```

Comment:
import this component like others, with `~`

Reply:
done